packages/frontend/src/creation-form/index.tsx

```diff
@@ -174,14 +174,14 @@ export const Component = ({
                         constraintValues[1] || 0n,
                     ],
                 );
-                console.log("initData", { initializationData });
                 return {
                     data: initializationData,
                     value: 0n,
                 };
             };
         }
 
+        console.log("ONCHANGE", timestamp);
```

Comment:
Left a `console.log` here.

Reply:
Addressed in e048b99f87c16ddc60ccbfc447324d2cdd6bb221